config.py

```diff
@@ -475,6 +475,33 @@
             '.*[0-9]+.*',
         ],
     ),
+    
+    fi = dict(
+        lang_name = 'Suomi',
+        lang_dir = 'ltr',
+        database = 'fiwiki_p',
+        wikipedia_domain = 'fi.wikipedia.org',
+
+        beginners_link = 'https://fi.wikipedia.org/wiki/Wikipedia:Merkitse_l%C3%A4hteet',
+        beginners_link_title = 'Wikipedia:Merkitse lähteet',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+
+        citation_needed_category = 'Lähteettömät_artikkelit',
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Lähde',
```

Comment:
https://fi.wikipedia.org/w/index.php?title=Toiminnot:T%C3%A4nne_viittaavat_sivut/Malline:L%C3%A4hde&hidetrans=1&hidelinks=1

So, "Lähde?" and "Fact" are the templates that redirects to "Lähde".